packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -459,6 +475,11 @@ export enum CompressionAlgorithms {
 	lz4 = "lz4",
 }
 
+/**
+ * Default count of reserved Ids that the runtime IdCompressor will initialize itself with if enabled.
+ */
+export const defaultIdCompressorReservedIdCount = 10;
```

Comment:
@justus-camp upon revisiting this, we should remove reservedIds from the API and not expose them. I plan on removing them in the future.

Reply:
ReservedIds have been removed